benchmark/attention_backend/NVIDIA_H200.md

```diff
@@ -0,0 +1,308 @@
+## Experiment - bench_serving | DeepSeek-V3
+
+| Configuration | Input token throughput (tok/s) | Output token throughput (tok/s) |
+|-------------|----------------|----------------|
+| attn=fa3,random_input_len=1024,random_output_len=1024 | 2216.96    | 2268.68    |
+| attn=flashinfer,random_input_len=1024,random_output_len=1024 | 2226.81    | 2278.76    |
+| attn=fa3,random_input_len=8192,random_output_len=1024 | 8369.41    | 1078.81    |
+| attn=flashinfer,random_input_len=8192,random_output_len=1024 | 8373.39    | 1079.32    |
+| attn=fa3,random_input_len=8192,random_output_len=8192 | 1385.71    | 1482.43    |
+| attn=flashinfer,random_input_len=8192,random_output_len=8192 | 1418.84    | 1517.88    |
+
+### Commands
+
+#### attn=fa3,random_input_len=1024,random_output_len=1024
+
+```
+python3 -m sglang.launch_server --attention-backend fa3 --tp 8 --model-path /shared/public/elr-models/deepseek-ai/DeepSeek-V3/1d044fd82b15f1cedb197a288e50cc96a2c27205
+python3 -m sglang.bench_serving --backend sglang --dataset-name random --request-rate 16 --num-prompts 200 --random-input-len 1024 --random-output-len 1024
```

Comment:
The default random ratio is 0, so the isl/osl is not 1024. The others are similar.